api/app.ts

```diff
@@ -18,11 +18,8 @@ instance.use(cors({ origin: '*', }));
 instance.use(express.json({ limit: '50mb' }));
 instance.use(express.urlencoded({ extended: true }));
 
-useRouter(instance, '/', article, true);
-useRouter(instance, '/', category, true);
-
-instance.use('/article.create', authRoute);
-instance.use('/article.create_raw', authRoute);
+useRouter(instance, article, true);
+useRouter(instance, category, true);
```

Comment:
If all the routes run though the middleware anyway. We can remove the third argument...